platform_versioned_docs/version-24.1/getting-started/proteinfold.mdx

```diff
@@ -0,0 +1,481 @@
+
+    ```python 
+    # Replace with the actual paths to your AlphaFold2 and ColabFold PDB files
+    alphafold_pdb = "/path/to/your/alphafold/output.pdb"
+    colabfold_pdb = "/path/to/your/colabfold/output.pdb"
+
+    # Create a dictionary of files
+    pdb_files = {
+        "AlphaFold": alphafold_pdb,
+        "ColabFold": colabfold_pdb
+    }
+
+    print("File paths set up successfully.")
+    ```
+
+1. Display file information:
+
+    ```python
+    display(HTML("<h3>Protein Structure Prediction Output Files:</h3>"))
+    for method, file_path in pdb_files.items():
+        if os.path.exists(file_path):
+            display(HTML(f"<p><strong>{method}:</strong> {file_path}</p>"))
+        else:
+            display(HTML(f"<p><strong>{method}:</strong> File not found at {file_path}</p>"))
+    ```
+
+1. Visualize structures:
+
+    ```python
+    valid_pdb_files = {method: file_path for method, file_path in pdb_files.items() if os.path.exists(file_path)}
+
+    if valid_pdb_files:
+        display(HTML("<h3>Protein Structure Visualization:</h3>"))
+        comparison = compare_proteins(valid_pdb_files)
+        display(comparison)
+    else:
+        display(HTML("<p>No valid PDB files found. Please check the file paths and ensure that the files exist.</p>"))
+    ```
+
+1. Add interactive elements:
+
+    ```python
+    if valid_pdb_files:
+        method_dropdown = widgets.Dropdown(
+            options=[method for method, file in valid_pdb_files.items()],
+            description='Select method:',
+            disabled=False,
+        )
+        
+        info_output = widgets.Output()
+        
+        def on_change(change):
+            with info_output:
+                info_output.clear_output()
+                selected_method = change['new']
+                selected_file = valid_pdb_files[selected_method]
+                print(f"Selected method: {selected_method}")
+                print(f"File path: {selected_file}")
+                print(f"File size: {os.path.getsize(selected_file) / 1024:.2f} KB")
+        
+        method_dropdown.observe(on_change, names='value')
+        
+        display(HTML("<h3>Structure Information:</h3>"))
+        display(widgets.VBox([method_dropdown, info_output]))
+    ```
+
+1. Display usage instructions:
+
+    ```python
+    display(HTML("""
+    <h3>How to use this visualization:</h3>
+    <ul>
+        <li>The protein structures from AlphaFold and ColabFold are shown side-by-side above.</li>
+        <li>You can interact with each structure independently:</li>
+        <ul>
+            <li>Click and drag to rotate the structure.</li>
+            <li>Scroll to zoom in and out.</li>
+            <li>Right-click and drag to translate the structure.</li>
+        </ul>
+        <li>Use the dropdown menu to select a specific method and view its file information.</li>
+    </ul>
+    """))
+    ```
+
+![Protein structure visualization](./_images/protein-structure-visualization.gif)
```

Comment:
Works on my machine :) 
![image](https://github.com/user-attachments/assets/6a5e84bd-43f6-42b5-b658-370a17d10cf6)

It may just have taken a while to load because the GIFs really make the page huge. We'll be swopping these out for MP4 video soon when we set up MUX for video hosting after Summit